backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/NormService.java

```diff
@@ -418,9 +449,103 @@ private List<NormExpressionEli> collectRelevantExistingExpressions(
       .toList();
   }
 
-  @SuppressWarnings("java:S125") // for the commented-out lines
-  private Zielnorm createZielNormen(final Zielnorm zielnorm) {
-    // For now just returning the same list but manually setting all to created
+  private Zielnorm createZielNormen(final Norm verkuendungNorm, final Zielnorm zielnorm) {
+    final AmendedNormExpressions amendedNormExpressions = verkuendungNorm
+      .getRegelungstext1()
+      .getMeta()
+      .getOrCreateProprietary()
+      .getOrCreateCustomModsMetadata()
+      .getOrCreateAmendedNormExpressions();
+
+    zielnorm
+      .expressions()
+      .forEach(expression -> {
+        if (expression.isCreated()) {
+          final Norm norm = loadNormPort
+            .loadNorm(new LoadNormPort.Options(expression.normExpressionEli()))
+            .orElseThrow(() ->
+              new IllegalStateException(
+                String.format("Norm %s must exist", expression.normExpressionEli())
+              )
+            );
+          if (expression.isGegenstandslos()) {
+            // Create new manifestation and set metadatum to gegenstandslos (date of announcement)
+            final Norm newManifestation = createNewVersionOfNormService.createNewManifestation(
+              norm
+            );
+            // Use announcement date of amending law (part of the work eli)
+            newManifestation.setGegenstandlos(
+              verkuendungNorm.getRegelungstext1().getMeta().getFRBRWork().getFBRDate()
+            );
+            updateOrSaveNormPort.updateOrSave(new UpdateOrSaveNormPort.Options(newManifestation));
+          }
```

Comment:
do we actually need this part? This should be covered by updating the previous expressions of the non-gegenstandslose expressions